pkgs/unified_analytics/test/event_test.dart

```diff
@@ -2,29 +2,328 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
+import 'dart:mirrors';
+
```

Comment:
@bkonyi i know using this package is a little "iffy", but in the context of this test file, i needed a way to ensure that each constructor was being tested from the `Event` class.

Thoughts?

Reply:
Ehhhhhh, I wouldn't encourage use of `dart:mirrors` but I don't have an alternative for you. If this functionality works, you're probably fine to use it, but lots of `dart:mirrors` has been left to bit-rot and won't work with newer language features (anything >= Dart 2.0 could be partly broken or might just not work).

There might be a way to do this with package that utilizes code gen, but I'm not sure.